src/components/home/Section.tsx

```diff
@@ -1,13 +1,18 @@
 // Package dependencies
 import React from 'react';
-import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { whiteContainer } from '@globals/styles';
+import { reactChildren } from '@globals/types';
+
+interface propTypes {
```

Comment:
We might want to come up with a naming convention for these interfaces at some point.